src/components/sidebar/components/sidebar-headings/sidebar-headings.module.css

```diff
@@ -13,3 +13,91 @@
 	padding-left: 8px;
 	padding-right: 8px;
 }
+
+.brandedRoot {
+	/* Generic default theme, eg. for HCP and Sentinel */
+	--gradient-start: var(--token-color-palette-neutral-100);
+	--gradient-stop: var(--token-color-palette-neutral-50);
+	--inset-border-color: var(--token-color-palette-neutral-200);
+
+	/* Set up themed property helpers */
+	--inset-border-shadow: inset 0 0 0 1px var(--inset-border-color);
+	--gradient-background: linear-gradient(
+		315deg,
+		var(--gradient-start) 0%,
+		var(--gradient-stop) 100%
+	);
+
+	align-items: center;
+	background: var(--gradient-background);
+	border-radius: 5px;
+	box-shadow: var(--inset-border-shadow);
+	color: var(--token-color-foreground-primary);
+	display: flex;
+	gap: 8px;
+	margin-bottom: 2px;
+	padding: 8px;
+	position: relative;
+	width: 100%;
+	z-index: 0;
```

Comment:
why is `z-index` needed here? 🤔

Reply:
This is a exact copy of the styling from `sidebar-nav-highlight-item.module` with the focus styling removed.